features/gba-dev/uitsluiting-kiesrecht-gba.feature

```diff
@@ -0,0 +1,75 @@
+	  | aanduiding uitgesloten kiesrecht (38.10) |
+		| A                                        |
+		Als gba personen wordt gezocht met de volgende parameters
+		| naam                | waarde                          |
+		| type                | RaadpleegMetBurgerservicenummer |
+		| burgerservicenummer | 000000139                       |
+		| fields              | uitsluitingKiesrecht            |
+		Dan heeft de response een persoon met de volgende 'uitsluitingKiesrecht' gegevens
+    | naam                    | waarde   |
+    | uitgeslotenVanKiesrecht | true     |
+
+	Rule: Uitsluiting van kiesrecht wordt alleen opgenomen wanneer de einddatum uitsluiting in de toekomst ligt
+		- Wanneer alleen het jaar van de einddatum uitsluiting bekend is, dan wordt de uitsluiting opgenomen tot en met dat jaar.
+		- Wanneer het jaar en de maand van de einddatum uitsluiting bekend is, en de dag niet, dan wordt de uitsluiting opgenomen tot en met die maand.
+		- Een volledig onbekende einddatum uitsluiting wordt hetzelfde geïnterpreteerd en weergegeven als het niet aanwezig zijn van die datum.
+		- Wanneer einddatum uitsluiting geen waarde heeft en uitsluiting kiesrecht wel, wordt uitsluiting kiesrecht wel opgenomen.
+		- Wanneer einddatum uitsluiting in het verleden ligt of vandaag is, wordt het niet opgenomen.
+
+		Abstract Scenario: kiesrecht <omschrijving>
+#   Functioneel zou dit moeten zijn :
+#   Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'kiesrecht' gegevens
+#   De kiesrechtgegevens zijn opgenomen in de lo3_pl tabel. Het toevoegen van kiesrecht gegevens zou in de automation code een update vereisen.
+#   Die update is nog niet geïmplementeerd, dus worden de kiesrecht met de insert van de inschrijving meegegeven.
+
+			Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'inschrijving' gegevens
+			| aanduiding uitgesloten kiesrecht (38.10) | einddatum uitsluiting kiesrecht (38.20) |
+			| <uitsluiting kiesrecht>                  | <einddatum uitsluiting>                 |
+			Als gba personen wordt gezocht met de volgende parameters
+			| naam                | waarde                          |
+			| type                | RaadpleegMetBurgerservicenummer |
+			| burgerservicenummer | 000000140                       |
+			| fields              | burgerservicenummer,uitsluitingKiesrecht |
+			Dan heeft de response een persoon met de volgende gegevens
+			| naam                                         | waarde                    |
+			| burgerservicenummer                          | 000000140                 |
+			| uitsluitingKiesrecht.uitgeslotenVanKiesrecht | <uitgeslotenVanKiesrecht> |
+			| uitsluitingKiesrecht.einddatum               | <einddatum>               |
+
+				Voorbeelden:
+		  | omschrijving                             | uitsluiting kiesrecht | einddatum uitsluiting | uitgeslotenVanKiesrecht | einddatum    |
```

Comment:
uitsluiting kiesrecht heeft altijd de waarde A. Dan is het niet nodig om het in de voorbeeld kolom op te nemen en kan het direct in de gegeven stap. Scheelt een kolom en scrollen naar rechts om alles te kunnen lezen